app/src/main/java/com/example/fivenightsescape/MapsActivity.kt

```diff
@@ -28,13 +32,33 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
     private lateinit var fusedLocationClient: FusedLocationProviderClient
     private lateinit var mMap: GoogleMap
 
+    //timer and lv things
+    private var levelChanger = 4 //levelChanger * delay = every how many milliseconds next lv, it changes every two levels
```

Comment:
Comment too long:
`MaxLineLength - [levelChanger] at /home/runner/work/FiveNightsEscape/FiveNightsEscape/app/src/main/java/com/example/fivenightsescape/MapsActivity.kt:36:5` keep it to be around 80 to 120 characters, ( or you can put it above the variable )